app/controllers/notes_controller.rb

```diff
@@ -2,19 +2,32 @@
 class NotesController < ApplicationController
   def create
 
-    if params[:service_id]
-      note = service.notes.build
-      note.note = params[:note][:note]
-      service.save!
-    elsif params[:resource_id]
-      note = resource.notes.build
-      note.note = params[:note][:note]
-      resource.save!
+    note = nil
+    note_json = params[:note]
+    if note_json[:service_id]
+      note = Note.create(service_id: note_json[:service_id], note: note_json[:note])
+      note.save!
```

Comment:
I the `save` is redundant, since `create` will already make a DB request, though you might want `create!` instead of `create` so that it throws an error if the data is invalid.